ntbs-service/wwwroot/css/header.scss

```diff
@@ -48,3 +50,16 @@
 .header-expand-menu-button {
     color: black;
 }
+
+.breadcrumb-trail {
+    display: none;
+}
+
+@media (min-width: 48.0625em) {

```

Comment:
The breadcrumb takes up a lot of space on mobile so opted to only show it for now on header - can implement full behaviour when we add the breadcrumb trail unless anyone has any input!

Reply:
I think that's fine - we'll get into details as we implement the breadcrumbs themselves, as you say.